### ISSUE TITLE
VCF Export for Contacts

### ISSUE THREAD

**Issue by iaean** [owncloud/core#24245]
Integration of Card-/CalDav in OC Core via sabre/dav is awesome. :+1:

But, could you add support for the sabre [vcfexportplugin](http://sabre.io/dav/vcf-export-plugin/) for use with Contacts, please. As well as Calendars supports the [icsexportplugin](http://sabre.io/dav/ics-export-plugin/).

```
# Works
https://oc.example.org/remote.php/dav/calendars/<userName>/<calendarName>?export

# Doesn't work in 9.0.1
https://oc.example.org/remote.php/dav/addressbooks/<userName>/<addressbookName>?export
```

Thanks a lot
Best regards

**Comment by DeepDiver1975** [comment]
Already there

**Comment by DeepDiver1975** [comment]
https://github.com/owncloud/core/blob/master/apps/dav/lib/server.php#L92
https://github.com/owncloud/core/blob/master/apps/dav/lib/server.php#L101

**Comment by RealRancor** [comment]
@iaean 

> Doesn't work in 9.0.1
> https://oc.example.org/remote.php/dav/addressbooks/<userName>/<addressbookName>?export

The correct URL for this is:

``https://oc.example.org/remote.php/dav/addressbooks/users/<userName>/<addressbookName>?export``

**Comment by iaean** [comment]
@RealRancor, I tried this pattern, too. Doesn't work for me. I get:
```
curl -s -u $userName https://oc.example.org/remote.php/dav/addressbooks/users/$userName/default?export | head
This is the WebDAV interface. It can only be accessed by WebDAV clients such as the ownCloud desktop sync client.
```
vs.
```
curl -s -u $userName https://oc.example.org/remote.php/dav/calendars/$userName/default?export | head
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//SabreDAV//SabreDAV//EN
CALSCALE:GREGORIAN
...
```
Both, `default` calendar and addressbook exists and works...

@DeepDiver1975, is this feature available in 9.0.2? I run 9.0.1.

**Comment by DeepDiver1975** [comment]
The card Plugin was added later. I have to lookup the version
